packages/gallery/src/components/item/itemView.js

```diff
@@ -982,6 +983,26 @@ class ItemView extends GalleryComponent {
     return tabIndex;
   }
 
+  onAnchorFocus() {
+    if (
+      (isSiteMode() || isSEOMode()) &&
+      !utils.isMobile() &&
+      window.document &&
+      window.document.activeElement &&
+      this.props.shouldUseExperimentalFeature
```

Comment:
1. Decouple this method from `this`
2. Decouple the `changeActiveElementIfNeeded` from `this`
3. Move both methods to another file, which the condition inside the `if` is shared between them 
4. Make sure you pass only necessary arguments to these functions (don't pass the entire props). Make the new functions depend on the smallest dependency possible.